### PR TITLE
Update code styling rules

### PR DESCRIPTION
**Pull request by jshjohnson**
Update code styling rules

### CODE REVIEW

**Comment by tinovyatkin** [comment]
@jshjohnson can you run all files update after merging/closing pending PRs? It's will be lot of merge conflicts 😢

**Comment by jshjohnson** [comment]
> @jshjohnson can you run all files update after merging/closing pending PRs? It's will be lot of merge conflicts 😢

What do you mean?

**Comment by tinovyatkin** [comment]
@jshjohnson I mean add rules (they are great), but run them on whole codebase after merging pending PRs, otherwise if you merge that now into master `git` will not be able to merge automatically many of them.

File: package.json [hunk 1]
```diff
@@ -63,13 +63,14 @@
     "chai": "^4.2.0",
     "csso-cli": "^3.0.0",
     "cypress": "3.5.0",
-    "eslint": "^6.5.1",
+    "eslint": "^6.6.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.4.0",
```
Comment (tinovyatkin) [inline package.json]: This one also can be updated to 6.6

**Comment by jshjohnson** [comment]
> @jshjohnson I mean add rules (they are great), but run them on whole codebase after merging pending PRs, otherwise if you merge that now into master `git` will not be able to merge automatically many of them.

Ah sorry, I understand. Yeah, will do 👍

File: .eslintrc [hunk 2]
```diff
@@ -32,6 +32,23 @@
...
+          "[static-methods]",
+          "[properties]",
+          "[conventional-private-properties]",
+          "constructor",
+          "[methods]",
+          "[conventional-private-methods]"
+        ],
+        "accessorPairPositioning": "getThenSet"
+      }
     ]
```
Comment (tinovyatkin) [inline .eslintrc]: @jshjohnson `sort-class-members` is lovely ESLint plugin with one major drawback - no built-in autofix.
So, for pleasure of lazy developers (like me), I would suggest to do

```sh
npm i -D jscodeshift sort-class-members-codemod
```

and edit `lint-staged.config.js`:

```js
module.exports = {
  "*.js": [
    "jscodeshift -t node_modules/sort-class-members-codemod/index",
    "eslint --fix --quiet -f visualstudio",
    "git add"
  ]
  // rest of file here
};
```

Comment (jshjohnson) [reply .eslintrc]: Hmmm i'm sure it was auto-fixing for me 🤔

Comment (tinovyatkin) [reply .eslintrc]: Nope, unfortunately. But lint-staged can help.

Comment (tinovyatkin) [reply .eslintrc]: Ooops, my bad! They had a new version with autofix! :tada:

**Pull request merged**